src/EditorFeatures/CSharpTest/CodeActions/ConvertLocalFunctionToMethod/ConvertLocalFunctionToMethodTests.cs

```diff
@@ -555,7 +556,7 @@ void M()
             }
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertLocalFunctionToMethod)]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertLocalFunctionToMethod), WorkItem(35180, "https://github.com/dotnet/roslyn/issues/35180")]
```

Comment:
we don't genrally mere workitems with facts.  we put a separate attribute.

Reply:
I.e having 

```csharp
[Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertLocalFunctionToMethod)]
[WorkItem(35180, "https://github.com/dotnet/roslyn/issues/35180")]
...
```

?

Didn't know that was legal in C# :)